pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,538 @@
+    echo "${OPTION}=big5" >> ${OUTPUT_FILE}
+    echo "${OPTION}=dec8" >> ${OUTPUT_FILE}
+    echo "${OPTION}=cp850" >> ${OUTPUT_FILE}
+    echo "${OPTION}=hp8" >> ${OUTPUT_FILE}
+    echo "${OPTION}=koi8r" >> ${OUTPUT_FILE}
+    echo "${OPTION}=latin1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=latin2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=swe7" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ascii" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ujis" >> ${OUTPUT_FILE}
+    echo "${OPTION}=sjis" >> ${OUTPUT_FILE}
+    echo "${OPTION}=hebrew" >> ${OUTPUT_FILE}
+    echo "${OPTION}=tis620" >> ${OUTPUT_FILE}
+    echo "${OPTION}=euckr" >> ${OUTPUT_FILE}
+    echo "${OPTION}=koi8u" >> ${OUTPUT_FILE}
+    echo "${OPTION}=gb2312" >> ${OUTPUT_FILE}
+    echo "${OPTION}=greek" >> ${OUTPUT_FILE}
+    echo "${OPTION}=cp1250" >> ${OUTPUT_FILE}
+    echo "${OPTION}=gbk" >> ${OUTPUT_FILE}
+    echo "${OPTION}=latin5" >> ${OUTPUT_FILE}
+    echo "${OPTION}=armscii8" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ucs2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=cp866" >> ${OUTPUT_FILE}
+    echo "${OPTION}=keybcs2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=macce" >> ${OUTPUT_FILE}
+    echo "${OPTION}=macroman" >> ${OUTPUT_FILE}
+    echo "${OPTION}=cp852" >> ${OUTPUT_FILE}
+    echo "${OPTION}=latin7" >> ${OUTPUT_FILE}
+    echo "${OPTION}=utf8mb4" >> ${OUTPUT_FILE}
+    echo "${OPTION}=cp1251" >> ${OUTPUT_FILE}
+    echo "${OPTION}=utf16" >> ${OUTPUT_FILE}
+    echo "${OPTION}=utf16le" >> ${OUTPUT_FILE}
+    echo "${OPTION}=cp1256" >> ${OUTPUT_FILE}
+    echo "${OPTION}=cp1257" >> ${OUTPUT_FILE}
+    echo "${OPTION}=utf32" >> ${OUTPUT_FILE}
+    echo "${OPTION}=geostd8" >> ${OUTPUT_FILE}
+    echo "${OPTION}=cp932" >> ${OUTPUT_FILE}
+    echo "${OPTION}=eucjpms" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--collation-server" ]; then
+    echoit "  > Adding possible value   utf8mb4_0900_ai_ci for option '${OPTION}' to the final list..."
+    echo "${OPTION}=utf8mb4_0900_ai_ci" >> ${OUTPUT_FILE}	  
+  elif [ "${OPTION}" == "--completion-type" ]; then
+    echoit "  > Adding possible values 0, 1, 2 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--concurrent-insert" ]; then
+    echoit "  > Adding possible values 0, 1, 2 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-log-files-in-group" ]; then
+    echoit "  > Adding possible values 0,1,2,5,10 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=5" >> ${OUTPUT_FILE}
+    echo "${OPTION}=10" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--slave-type-conversions" ]; then
```

Comment:
Please note that from server version 8.0.26 and onwards, "slave" and "master" word has been deprecated. Instead upstream has renamed the variables. 
slave => replica
master => source
rename variable to replica-type-conversions

Reply:
Both replica-type-conversions and slave-type-conversions options are present in verbose